apps/code-examples/src/app/code-examples/indicators/alert/basic/alert-demo.component.html

```diff
@@ -1,8 +1,9 @@
 <sky-alert
-  alertType="info"
-  [closeable]="true"
+  [alertType]="days < 8 ? 'danger' : 'warning'"
+  [closeable]="days >= 8"
+  [descriptionType]="days < 8 ? 'danger' : 'important-warning'"
   (closedChange)="onClosedChange($event)"
-  descriptionType="important-info"
+  data-sky-id="alert-demo"
```

Comment:
Native attributes should be placed at the "top" of the element.

Reply:
Done